utils/uiTypes/proposalCreationTypes.ts

```diff
@@ -391,6 +392,18 @@ export interface MangoMakeChangeReferralFeeParams2 {
   refMngoRequired2: number
 }
 
+export interface PsyFinanceMintAmericanOptionsForm {
+  contractSize: number
+  expirationUnixTimestamp: number
+  optionTokenDestinationAccount: string
+  quoteMint: string
+  size: number | undefined
+  strike: number | undefined
```

Comment:
In the instruction form, `strike` is set to `0` by default. I believe it never can be `undefined` then